modules/xmpp/JingleSessionPC.js

```diff
@@ -328,6 +328,8 @@ export default class JingleSessionPC extends JingleSession {
         pcOptions.enableInsertableStreams = options.enableInsertableStreams;
         pcOptions.videoQuality = options.videoQuality;
         pcOptions.forceTurnRelay = options.forceTurnRelay;
+        pcOptions.stereo = options.stereo;
```

Comment:
I have a minor suggestion, will you be able to add these two under a `audioQuality` setting in config.js like how we have video quality settings ?

Reply:
IIRC, @saghul and I talked about this and said we'd initially add it as hardcoded and then perhaps switch to a configurable object. I have no issue with doing it now, WDYT, Saul?

Reply:
Yeah, let's group them in audioQuality, me likey!